src/entt/core/tuple.hpp

```diff
@@ -100,4 +101,16 @@ forward_apply(Func) -> forward_apply<std::remove_reference_t<std::remove_cv_t<Fu
 
 } // namespace entt
 
+template <typename... Type>
+struct std::tuple_size<entt::type_list<Type...>>: std::integral_constant<std::size_t, entt::type_list<Type...>::size> {};
```

Comment:
Not sure this is the right place for these. 🤔 
`type_traits.hpp` seems more appropriate actually.

Reply:
Well, that was your suggestion back then ^^

Reply:
Yeah, I know 😅 I changed my mind though, it wasn't a good suggestion actually.